lib/trento.pm

```diff
@@ -767,12 +767,13 @@ Get the api-key from the Trento installation
 =cut
 
 sub trento_api_key {
-    my ($wd, $basedir) = @_;
-    my $cmd = join(' ', $basedir . '/trento-server-api-key.sh',
+    my ($basedir) = @_;
+    my $cmd = join(' ', $basedir . '/trento_deploy/trento_deploy.py',
+        '--verbose', 'api_key',
         '-u', 'admin',
         '-p', get_trento_password(),
         '-i', get_trento_ip(),
-        '-d', $wd, '-v');
+        '-v');
```

Comment:
I think that this last `-v` is not needed too

Reply:
thx for the comment master, will remove the argument.